resources/carrier/en/44.txt

```diff
@@ -13,6 +13,108 @@
 # limitations under the License.
 
 # Carrier details updated from http://www.ofcom.org.uk/static/numbering/S7.xls
+# Generated on 2017-07-14.
+
+# Carrier names have been updated to friendlier trading/brand names.
+# The mapping of company names can be found in the project below:
+#  - (aq) Limited trading as aql: aql
+#  - 08Direct Limited: 08Direct
+#  - 09 Mobile Ltd: 09 Mobile
+#  - 24 Seven Communications Ltd: 24 Seven
+#  - AMSUK LTD: AMSUK
+#  - Ace Call Limited: Ace Call
+#  - Airwave Solutions Ltd: Airwave
+#  - Alliance Technologies LLC: Alliance
+#  - Andrews & Arnold Ltd: Andrews & Arnold
+#  - Anywhere Sim Limited: Anywhere Sim
+#  - BT OnePhone Limited: BT OnePhone
+#  - Bellingham Telecommunications Limited: Bellingham
+#  - CFL Communications Limited: CFL
+#  - Cheers International Sales Limited: Cheers
+#  - Citrus Telecommunications Ltd: Citrus
+#  - Cloud9 Communications Limited: Cloud9
+#  - Cloud9 Mobile Communications Ltd: Cloud9
+#  - Compatel Ltd: Compatel
+#  - Confabulate Limited: Confabulate
+#  - Core Communication Services Ltd: Core Communication
+#  - Core Telecom Limited: Core Telecom
+#  - Dynamic Mobile Billing Limited: Oxygen8
+#  - EE Limited ( TM): EE
+#  - EE Limited (Orange): Orange
+#  - Esendex Limited: Esendex
+#  - FleXtel Limited: FleXtel
+#  - Fogg Mobile AB: Fogg
+#  - Gamma Telecom Holdings Ltd: Gamma Telecom
+#  - Global Reach Networks Limited: GlobalReach
+#  - Globecom International Limited.: Globecom
+#  - Guernsey Airtel Limited: Airtel
+#  - HAY SYSTEMS LIMITED: HSL
+#  - Hanhaa Limited: Hanhaa
+#  - Hutchison 3G UK Ltd: Three
+#  - IPV6 Limited: IPV6
+#  - IV Response Limited: IV Response
+#  - Icron Network: Icron
+#  - Icron Network Limited: Icron Network
+#  - JT (Guernsey) Limited: JT
+#  - JT (Jersey) Limited: Jersey Telecom
```

Comment:
I had just copied this over from the old data, but I think this should just be JT or Join Together (Source: http://www.jtglobal.com/Jersey/)